msg/Arrays.msg

```diff
@@ -0,0 +1,34 @@
+# Arrays of different types
+bool[3] bool_values
+byte[3] byte_values
+char[3] char_values
+float32[3] float32_values
+float64[3] float64_values
+int8[3] int8_values
+uint8[3] uint8_values
+int16[3] int16_values
+uint16[3] uint16_values
+int32[3] int32_values
+uint32[3] uint32_values
+int64[3] int64_values
+uint64[3] uint64_values
+string[3] string_values
+BasicTypes[3] basic_types_values
+Constants[3] constants_values
+Defaults[3] defaults_values
+bool[3] bool_values_default [true, false, true]
```

Comment:
The current fixture uses `[false, true, false]`.

Reply:
90395df